### ISSUE TITLE
L’élection n’est pas un outils démocratique

### ISSUE THREAD

**Issue by niok35** [Philaeux/VIrepublique2017#9]
L’élection n’est pas un outils démocratique

**Aristote**: «Il est démocratique, par exemple, de tirer les magistrats au sort ; oligarchique, de les élire »
**Rousseau**: «Spartes était une aristocratie parce que ses magistrats étaient élus »
**Montesquieu**: « Le suffrage par le sort est de la nature de la démocratie ; le suffrage par le choix est de celle de l’aristocratie »

L’élection a une composante démocratique car tout le monde vote (suffrage universel)
Mais l’élection a une composante OLIGARCHIQUE car on vote pour des gens pré-selectionnés par l’argent (budget de campagne en dizaine de millions d’euro) et par l’accès aux Medias (Bolloré, Bouygues, Drahi, Lagardère, Dassaut).

Seul un candidat massivement financé sera élu.

Chercher sur internet, la corrélation qui existe entre 
- le nombre de vote Vs budget de campagne 
- le nombre de vote Vs temps de parole.

Plus le temps de parole est élevé, plus le candidat gagne des voix
Plus le budget de campagne est élevé (nombreux meeting, tracts,...), plus le candidat gagne de voix

Le temps de parole TV est mesuré par le CSA (entièrement nommé par le président de l'exécutif !)
Le temps de parole est ÉQUITABLE et non égalitaire. Plus un partie est important plus son candidat peut parlé. En 2007, Sarkosy a parlé 60 fois plus que Dupont Aignan !
Qu'est ce qu'un partie important ? Son nombre d'adhérent ? Qui connait le nombre d'adhérent du PS ou des républicains ?

Vous êtes attaché à l'élection malgré ses défauts... ok. Élisez un président de l'exécutif... mais affaiblissez/contrôlez ses pouvoirs.

Utilisons le tirage au sort de volontaire (majeur, sans casier judiciaire,...) pour l'assemblée législative.
Avantages:
- Vrai séparation entre l'exécutif et les tirés au sort.
- Changement régulier de l'assemblée = rotation des charges = principe fort de la démocratie.
- Qui dit changement régulier, dit corruption plus difficile.
- Il y aura moins de lois et elles seront plus simple et plus facilement applicable.
Inconvénients:
- Non professionnel: Il faut former les gens au début de leur mandat.
- Quelques fachos dans l'assemblée: Sur le nombre, ils seront minoritaires.
- Quelques crétins dans l'assemblée: Idem, ils seront minoritaires.
- Incompétents: Un représentant est justiciable. Si il fait des lois contre le peuple, il risque amendes/prison/révocation. Une personne tirée doit être volontaire... sachant qu'elle prend des responsabilité et qu'elle sera jugée.... ça limite les crétins volontaires.

Le tirage au sort est un outil démocratique... Comme tout outil, il faut bien l'utiliser.

**Comment by aloisdg** [comment]
Démocratie Liquide FTW.

**Comment by niok35** [comment]
"Démocratie Liquide"
Quand je vois qu'à Rennes (plus de 200 000 habitants), on se retrouve à seulement 8 personnes au groupe démocratie de Nuit Debout.
Quand je vois qu'il est difficile de discuter/motiver les gens pour parler démocratie et/ou politique.
Quand je vois le temps important qu'il faut prendre pour être bien informé, pour monter en compétence sur un sujet... pour pouvoir prendre une décision (= voter une loi)

Je me dis que la démocratie direct et liquide sont juste impossible.
On n'a clairement pas le temps et/ou l'envie de travailler pour faire fonctionner la démocratie liquide ou direct.

Je suis pour une démocratie représentative, avec une rotation des charges (pas de cumul des mandats, un seul mandat, casier judiciaire vierge, non professionnel de la politique), des contres-pouvoirs (révocabilité des élus, justiciables), avec une séparation des pouvoirs, avec des médias indépendants, avec une transparence des comptes et des prises de décision, indépendance monétaire,...

**Comment by aloisdg** [comment]
@niok35 vrai. Je pense que tout le monde est intéressé par le sujet, mais que personne ne lui accorde suffisamment de temps. Lorsque le travail sera enfin une rareté, nous aurons bien plus de temps.

**Comment by Philaeux** [comment]
Dans la forme actuelle du programme, le peuple peut prendre part aux décisions de l'assemblée directement. Je pensais par exemple à faire 75% des votes de l'assemblées venant des députés, et 25% directement du peuple. Les technologies actuelles nous le permettent.

Dans une forme plus ancienne du programme, je souhaitais rajouter une 3e chambre dans le système de la Ve république. Cette chambre aurait pour vocation de discuter en parallèle des décisions de l'assemblée nationale. En somme, une assemblée nationale jumelée à une assemblée populaire. Cette seconde assemblée est renouvelée tous les ans (pendant les vacances d'été), tirée au sort parmi la population éligible (le peuple), et guidée par des experts +/- techniques et apolitiques.

Je ne souhaite pas "détruire" l'assemblée dans sa forme actuelle même si les débats qui y ont lieu sont... questionnables. Que pensez vous de ces 2 versions?

**Comment by Grenadingue** [comment]
Je pense que pour une VI ème république, l’idéal serait de repartir from scratch; d'abolir complètement le système actuel qui n'est qu'une illusion de démocratie. La [démocratie liquide](https://fr.wikipedia.org/wiki/D%C3%A9mocratie_liquide) suggérée par @aloisdg est pour moi un élément extrêmement intéressant, car en plus de fournir __un vrai outil démocratique__, celui-ci, une fois son apprentissage fait par la population fourni a chaque citoyen une vraie __responsabilité politique__, que chacun à la __possibilité d'assumer, ou de déléguer__. Ainsi une veritable voix peut être donnée a chacun. Cependant, je ne pense pas que cela empêche en parallèle la mise en place d'un couple du type sénat/assemblée, afin d'avoir des gens (élus par démocratie liquide) qui travaillent a temps plein pour faire de la politique. Enfin, étant donne le nombre de personnes en France, la démocratie liquide risque de montrer des failles concernant la possibilité que plusieurs choix ou personnes puissent être élues sans être départagées. Ici donc pourrait entrer en jeu le principe de choix aléatoire suggéré par @niok35
